packages/server/src/oauth/authorize.ts

```diff
@@ -78,35 +78,35 @@ async function validateAuthorizeRequest(req: Request, res: Response, params: Rec
   // Then, validate all other parameters.
   // If these are invalid, redirect back to the redirect URI.
   if (!params.scope) {
-    sendErrorRedirect(res, redirectUri, 'invalid_request', state);
+    sendErrorRedirect(res, redirectUri, 'invalid_request', 'Missing scope', state);
     return false;
   }
   if (params.response_type !== 'code') {
-    sendErrorRedirect(res, redirectUri, 'unsupported_response_type', state);
+    sendErrorRedirect(res, redirectUri, 'unsupported_response_type', 'Invalid response type', state);
     return false;
   }
   if (params.request) {
-    sendErrorRedirect(res, redirectUri, 'request_not_supported', state);
+    sendErrorRedirect(res, redirectUri, 'request_not_supported', 'Missing request', state);
```

Comment:
This seems inverted, request is not missing in this case but seems like it's just not allowed?